base/kustomization.yaml

```diff
@@ -17,10 +17,20 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - 200-clusterrole-backend.yaml
+  - 200-clusterrole-extensions.yaml
   - 200-clusterrole-pipelines.yaml
+  - 200-clusterrole-tenant.yaml
   - 200-clusterrole-triggers.yaml
-  - 201-clusterrolebinding.yaml
+  - 201-clusterrolebinding-backend.yaml
+  - 201-clusterrolebinding-extensions.yaml
+  - 201-clusterrolebinding-tenant.yaml
+  # this one should be used instead of 201-clusterrolebinding-extensions.yaml
```

Comment:
So what's the purpose in these comments still being here, are users expected to comment this part out accordingly (going against the way we use kustomize in general with base/overlays here, or maybe left behind as a mistake and can now be removed?)

Reply:
It’s here as a comment to help review. I didn’t write overlays for single namespace yet.
I am not completely sure how to design base/overlays as the resources you pointed out are mutually exclusive.
Two bases ?
I think I will make a dedicated PR for that, I am not sure it’s a good idea to do it here.